CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contribute
+
+CARTO VL platform is an open-source library. We are more than happy to receive your contributions to the code and the documentation as well.
+
+## Filling a ticket
+
+If you want to open a new issue in our repository, please follow these instructions:
+
+- Add a descriptive title.
+- Write a good description, answering the following questions:
+    - what is happening?
+    - what should happen?
```

Comment:
"What should be happening instead?" (to remain consistent with the previous question)